sdk/ml/azure-ai-ml/azure/ai/ml/entities/_builders/data_transfer_func.py

```diff
@@ -0,0 +1,263 @@
+# ---------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# ---------------------------------------------------------
+from typing import Optional, Dict, Union
+
+from azure.ai.ml.entities._component.datatransfer_component import DataTransferCopyComponent, \
+    DataTransferImportComponent, DataTransferExportComponent
+from azure.ai.ml.constants._common import AssetTypes
+from azure.ai.ml.constants._component import ComponentSource, ExternalDataType, DataTransferBuiltinComponentUri, \
+    DataTransferTaskType
+from azure.ai.ml.entities._inputs_outputs.external_data import Database, FileSystem
+from azure.ai.ml.entities._inputs_outputs.output import Output
+from .data_transfer import DataTransferCopy, DataTransferImport, DataTransferExport, _build_source_sink
+from .command_func import _parse_inputs_outputs, _parse_input, _parse_output
+
+
+def copy_data(
```

Comment:
seems most of the logics are shared?

Reply:
yeah, datatransfer need three builders for corresponding task.